exporter/loggingexporter/logging_exporter.go

```diff
@@ -165,7 +165,7 @@ func (b *logDataBuffer) logDoubleHistogramDataPoints(ps pdata.DoubleHistogramDat
 
 		buckets := p.BucketCounts()
 		if len(buckets) != 0 {
-			for _, bucket := range buckets {
+			for i, bucket := range buckets {
```

Comment:
Good catch, do you mind changing this to "j" to avoid shadowing the previous variable, and make it clear that refers to this for loop?

Reply:
Yup, will do.